man/man5/zfs-module-parameters.5

```diff
@@ -2385,6 +2385,19 @@ vdev while discarding the checkpoint.
 Default value: \fB16,777,216\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_special_class_metadata_reserve_pct\fR (int)
+.ad
+.RS 12n
+Only put small data blocks onto special allocation vdevs when the free space
+on said vdevs exceeds this number. This ensures space is available for pool
+metadata as the special vdevs approach capacity.
```

Comment:
What do you think of this slightly modified phrasing:

"Only allow small data blocks to be allocated on the special and dedup vdev types when the available free space percentage on these vdevs exceeds this value.  This ensures reserved space is available for pool meta data as the special vdevs approach capacity."